src/Promact.TestCaseManagement.DomainModel/Enums/Condition.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Promact.TestCaseManagement.DomainModel.Enums
+{
+    public enum Condition
+    {
+        PreCondition,
```

Comment:
Ok, If you discuss requirement with someone other than me then you are suppose to convey that via email. I don't have an idea that you discussed something with someone else and that is why you have created model like this. If I had known requirement update I would have not spent much of the time reviewing it. It is very important that you keep me in a loop.